src/Transport/Receiving/MessageReceiverNotifier.cs

```diff
@@ -105,7 +108,10 @@ await Task.WhenAll(tasks)
                 logger.Error("The receiver failed to stop with in the time allowed (30s)");
             }
 
-            await completion.Complete().ConfigureAwait(false);
+            if (completion != null)
+            {
+                await completion.Complete().ConfigureAwait(false);
```

Comment:
Can't be expressed as `await completion?.Complete().ConfigureAwait(false);`?

Reply:
https://stackoverflow.com/questions/40436213/why-elvis-operator-doesnt-work-with-async-await